### ISSUE TITLE
System crashes / reboots at installation

### ISSUE THREAD

**Issue by Northwave** [HungerHa/libreelec_package_argonforty-device#11]
**Describe the bug**
title says it all - when trying to install from zip, after a few seconds i can see kodi loding screen, no add-on visible in add-ons

**To Reproduce**
1. Followed the install manual

**Expected behavior**
Add-on gets installed and is working

**Debug log**
https://paste.libreelec.tv/eminent-sunfish.log

**Screenshots**
not possible

**LibreELEC (please complete the following information):**
 - Device: RPi4
 - Version 12.0.0
 - ArgonOne v2

**Comment by HungerHa** [comment]
Hi @Northwave,

a reboot is unusual, even if the add-on would have an error in the Python scripts. But I could immediatly found another general issue in your hardware configuration:

Please check the power supply you are using. The provided log directs to undervoltage at 2 places in the log. Maybe this triggers the reboot, you should fix this first.

`RPi throttled=0x50005`

`Feb 27 18:26:05.004294 LibreELEC kernel: hwmon hwmon0: Undervoltage detected!`

**Comment by Northwave** [comment]
Hi @HungerHa 

Ah thx, I now connected the official power supply. still I never encountered any issues with the power supply I had used before.

the error seems a bit odd: https://paste.libreelec.tv/sensible-moccasin.log
last three lines in the log

saying its not a zip file ... i downloaded it by curl -O https://....

nevermind ... I had an issue downloading it ... it downloaded 0 bytes. I then used wget instead.

It now installed correctly. this can be closed now :)
